src/main/java/com/radixdlt/crypto/KeyHandler.java

```diff
@@ -34,7 +34,7 @@
 	 * @return An {@link ECDSASignature} with {@code r} and {@code s} values included
 	 * @throws CryptoException if the {@code privateKey} is invalid
 	 */
-	ECDSASignature sign(byte[] hash, byte[] privateKey) throws CryptoException;
+	ECDSASignature sign(byte[] hash, byte[] privateKey, boolean enforceLowS, boolean beDeterministic) throws CryptoException;

```

Comment:
`beDeterministic` sounds awkward to me, could just be `deterministic`

Reply:
Changed to `useDeterministicSignatures` :D also added java doc for the two boolean params.